packages/core/src/browser/view-container.ts

```diff
@@ -837,6 +857,20 @@ export class ViewContainer extends BaseWidget implements StatefulWidget, Applica
         return false;
     }
 
+    protected getSashHoverDelay(): void {
+        const sashHoverDelay = this.corePreferences.get(SASH_DELAY_PREFERENCE);
```

Comment:
The variables would not be necessary if we update the following to:

```suggestion
        const sashHoverDelay = this.corePreferences['workbench.sash.hoverDelay'];
```

```ts
const SASH_DELAY_PREFERENCE = 'workbench.sash.hoverDelay';
```